edgexfoundry/app-functions-sdk-go
# feat: Enable use of new messaging Command Client, fix template issues

close #1190

In Draft until bootstrap PR https://github.com/edgexfoundry/go-mod-bootstrap/pull/384 is merged

signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

Run non-secure EdgeX with latest WIP images


## Testing Instructions
Build template app and run with Device Virtual (latest Nexus image)
Verify no error when event is processed
Verify logs have the following (device name and command name will vary)
```
"Device Random-Boolean-Device has 4 commands to choose from. (in pipeline 'default-pipeline')"
"SendGetCommand successfully received new event from GET command Bool on Random-Boolean-Device device in pipeline 'default-pipeline'"
```

## New Dependency Instructions (If applicable)
